index.js

```diff
@@ -14,33 +13,33 @@ app.launch(function(req, res) {
   // destination (and validation)
   // --> all this would be much shorter if we use presets from a webapp.
   // --> can also use user location from the phone, but less accurate.
-
-  var prompt = 'With CARVIS you can order the cheapest or fastest ride available. Tell me CHEAPEST or FASTEST and your start and end destinations';
+  var prompt = 'With CARVIS you can order the cheapest or fastest car available. For example, you can say, CARVIS, find me the cheapest ride to Hack Reactor';
 
   // NOTE: in launch only for testing
-  rideHelper.placesCall('944 market st');
-  rideHelper.getEstimate('fastest');
+  rideHelper.placesCall('stanford university', function(dest) {
+    rideHelper.getEstimate('cheapest', dest);
```

Comment:
yes, probably from our db (user's home location) if a start location is not otherwise provided to alexa

for now im assuming a hardcoded home for testing purposes